src/controllers/Article.controller.js

```diff
@@ -0,0 +1,130 @@
+import { ArticleServive } from "../services";
+export default class Message {
```

Comment:
what is message doing if not about messages use the actual name "naming convation"